src/main/java/org/jabref/gui/util/StreamGobbler.java

```diff
@@ -0,0 +1,22 @@
+package org.jabref.gui.util;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.function.Consumer;
+
+public class StreamGobbler implements Runnable {
+
+    private InputStream inputStream;
+    private Consumer<String> consumer;
+
+    public StreamGobbler(InputStream inputStream, Consumer<String> consumer) {
+        this.inputStream = inputStream;
+        this.consumer = consumer;
+    }
+
+    @Override
+    public void run() {
+        new BufferedReader(new InputStreamReader(inputStream)).lines().forEach(consumer);
```

Comment:
Does this need to be closed? Do we need try-with-resources here? If not, please add a comment why the buffered reader is not closed.